influxql.go

```diff
@@ -187,7 +196,14 @@ func (n *InfluxQLNode) runBatchInfluxQL() error {
 				kind = reflect.Float64
 			}
 		} else {
-			k := reflect.TypeOf(b.Points[0].Fields[c.field]).Kind()
+			f, ok := b.Points[0].Fields[c.field]
+			if !ok {
+				n.incrementErrorCount()
+				n.logger.Printf("E! field %s missing from point, skipping batch\n", c.field)
```

Comment:
Small nit, but there is no need for a new line here as the logger will do that already.

Also it looks like you need to rebase as the CHANGELOG diff looks out of date.